code/game/jobs/job/civilian.dm

```diff
@@ -164,6 +164,35 @@
 	shoes = /obj/item/clothing/shoes/brown
 	l_ear = /obj/item/device/radio/headset/headset_cargo
 
+/datum/job/cargoofficer
+	title = "Supply Officer"
+	flag = CARSEC
+	department = "Cargo"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the quartermaster, head of personnel, and head of security"
+	selection_color = "#adadad"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_maint_tunnels, access_mailsorting, access_cargo, access_qm, access_mining, access_mining_station)
+	minimal_access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_mining, access_mining_station, access_maint_tunnels, access_cargo, access_cargo_bot, access_mailsorting)
```

Comment:
`minimal_access` has more access than `access`

We are running on `minimal_access` so make sure that they are either the same or `access` has actually more access than `minimal_access`